### PR TITLE
german translation (again)

### PR DESCRIPTION
**Pull request by Catchears**
# German translation yet again

I don't mean to be annoying by constantly opening a new PR as the old one is merged, just want to ensure there's a translation for newish features available.

To my fellow translators @end-me-please and @Shawak - I saw the Serpulo sector names and personally think it would be better to either find a translation for all of them or to leave them all English, not a half-half as we have now. I'd suggest the below (not committed yet), do you have any thoughts / things you would change?

I'm unsure about Ground Zero and Impact 0078, they seem like terminology / names as opposed to the other more descriptive names so I've left them as they were for now.
```
sector.impact0078.name = Impact 0078
sector.groundZero.name = Ground Zero
sector.craters.name = Die Krater
sector.frozenForest.name = Gefrorener Wald
sector.ruinousShores.name = Trümmerstrände
sector.stainedMountains.name = Geflecktes Gebirge
sector.desolateRift.name = Einsame Kluft
sector.nuclearComplex.name = Kernherstellungskomplex
sector.overgrowth.name = Überwucherung
sector.tarFields.name = Teerfelder
sector.saltFlats.name = Salzebenen
sector.fungalPass.name = Infizierter Gebirgspass
sector.biomassFacility.name = Biomassensyntheselabor
sector.windsweptIslands.name = Windgepeitschte Inseln
sector.extractionOutpost.name = Extraktionsaußenposten
sector.planetaryTerminal.name = Planetares Launchterminal
sector.coastline.name = Küstenlinie
sector.navalFortress.name = Wasserfestung
```
---
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] Translation PR.

### CODE REVIEW

File: core/assets/bundles/bundle_de.properties [hunk 1]
```diff
@@ -585,31 +585,34 @@ research.multiplayer = Nur der Host kann forschen.
...
-objective.buildcount.name = Build Count
-objective.unitcount.name = Unit Count
-objective.destroyunits.name = Destroy Units
-objective.timer.name = Timer
-objective.destroyblock.name = Destroy Block
-objective.destroyblocks.name = Destroy Blocks
-objective.destroycore.name = Destroy Core
-objective.commandmode.name = Command Mode
+
+objective.research.name = Erforsche
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:589]: Forschen

File: core/assets/bundles/bundle_de.properties [hunk 2]
```diff
@@ -585,31 +585,34 @@ research.multiplayer = Nur der Host kann forschen.
...
+objective.produce.name = Erstelle
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:590]: Herstellen

File: core/assets/bundles/bundle_de.properties [hunk 3]
```diff
@@ -585,31 +585,34 @@ research.multiplayer = Nur der Host kann forschen.
...
+objective.item.name = Erhalte
+objective.coreitem.name = Lagere im Kern
+objective.buildcount.name = Blöcke bauen
+objective.unitcount.name = Einheiten herstellen
+objective.destroyunits.name = Zerstöre Einheiten
+objective.timer.name = Zeit
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:596]: Vielleicht Zeitmesser? Ich kenne den Kontext leider nicht

File: core/assets/bundles/bundle_de.properties [hunk 4]
```diff
@@ -737,18 +741,19 @@ sector.planetaryTerminal.description = Das Endziel.\n\nDiese Uferbasis besitzt e
 
 sector.onset.name = Der Anfang
 sector.aegis.name = Aegis
-sector.lake.name = Name
-sector.intersect.name = Intersect
+sector.lake.name = See
+sector.intersect.name = Schnitt
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:745]: Schnittmenge?

File: core/assets/bundles/bundle_de.properties [hunk 5]
```diff
@@ -737,18 +741,19 @@ sector.planetaryTerminal.description = Das Endziel.\n\nDiese Uferbasis besitzt e
...
-sector.lake.description = This sector's slag lake greatly limits viable units. A hover unit is the only option.\nResearch the [accent]ship fabricator[] and produce an [accent]elude[] unit as soon as possible.
-sector.intersect.description = Scans suggest that this sector will be attacked from multiple sides soon after landing.\nSet up defenses quickly and expand as soon as possible.\n[accent]Mech[] units will be required for the area's rough terrain.
-sector.atlas.description = This sector contains varied terrain and will require a variety of units to attack effectively.\nUpgraded units may also be necessary to get past some of the tougher enemy bases detected here.\nResearch the [accent]Electrolyzer[] and the [accent]Tank Refabricator[].
-sector.split.description = The minimal enemy presence in this sector makes it perfect for testing new transport tech.
-sector.basin.description = {Temporary}\n\nThe last sector for now. Consider this a challenge level - more sectors will be added in a later release.
+sector.split.name = Teilung
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:747]: Teilen?

File: core/assets/bundles/bundle_de.properties [hunk 6]
```diff
@@ -1173,7 +1178,7 @@ rules.title.planet = Planet
 rules.lighting = Blitze
 rules.fog = Kriegsnebel
 rules.fire = Feuer
-rules.anyenv = <Any>
+rules.anyenv = <jede>
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1181]: Jede

**Review by Shawak — CHANGES_REQUESTED** [review]

File: core/assets/bundles/bundle_de.properties (hunk 1 above)
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:589]: Die `objective.xxx.name` Einträge sind zum Auswählen der Ziele eines Spiels, ich habe jetzt aus allen Infinitivformen gemacht.

File: core/assets/bundles/bundle_de.properties (hunk 2 above)
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:590]: Hier würde ich doch lieber bei Erstellen (infinitv) bleiben, da die `objective.xxx.name` Einträge zur Auswahl des Kartenziels da sind, und das Ziel `Erstelle: xxx` heißt, nicht `Stelle her: xxx`.

File: core/assets/bundles/bundle_de.properties (hunk 3 above)
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:596]: Ist dann bei der Zielauswahl einer Karte zu sehen und wartet einfach eine bestimmte Zeit, bevor es weitermacht. Der Englische Namen 'Timer' ist auch nicht viel deskriptiver. Ich würde es so lassen.

File: core/assets/bundles/bundle_de.properties (hunk 4 above)
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:745]: Generell ja, aber da es sich hier um den Namen eines Sektors handelt erscheint mir 'Schnittmenge' ein bisschen komisch.

File: core/assets/bundles/bundle_de.properties (hunk 5 above)
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:747]: Da es sich hier wieder um einen Namen handelt, würde ich eher bei Substantiven bleiben.

**Review by Shawak — APPROVED** [review]

File: core/assets/bundles/bundle_de.properties [hunk 7]
```diff
@@ -1672,8 +1672,8 @@ block.diffuse.name = Diffusor
 block.basic-assembler-module.name = Einfaches Verbesserermodul
 block.smite.name = Smite
 block.malign.name = Malin
-block.flux-reactor.name = Flux Reactor
-block.neoplasia-reactor.name = Neoplasia Reactor
+block.flux-reactor.name = Fluxreaktor
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1675]: Flux-Reaktor

File: core/assets/bundles/bundle_de.properties [hunk 8]
```diff
@@ -1672,8 +1672,8 @@ block.diffuse.name = Diffusor
...
+block.neoplasia-reactor.name = Neoplasmareaktor
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1676]: Neoplasma-Reaktor

**Review by Shawak — CHANGES_REQUESTED** [review]

**Review by Shawak — APPROVED** [review]

File: core/assets/bundles/bundle_de.properties [hunk 9]
```diff
@@ -711,16 +715,16 @@ sector.impact0078.name = Impact 0078
...
 sector.frozenForest.name = Gefrorener Wald
-sector.ruinousShores.name = Ruinous Shores
-sector.stainedMountains.name = Stained Mountains
-sector.desolateRift.name = Desolate Rift
-sector.nuclearComplex.name = Nuclear Production Complex
-sector.overgrowth.name = Overgrowth
+sector.ruinousShores.name = Trümmerstrände
+sector.stainedMountains.name = Geflecktes Gebirge
+sector.desolateRift.name = Einsame Kluft
+sector.nuclearComplex.name = Kernherstellungskomplex
```
[the remark this answers is not in the document]
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:721]: Gute Anmerkung - den Hinweis mit Thorium finde ich ein bisschen zu deutlich, und bei Nukleare Produktionsstätte klingt es mMn so, als sei die Produktionsstätte nuklear. Ich fände Nuklear-Produktionsstätte ein guter Kompromiss, was hältst du davon?

[the remark this answers is not in the document]
Comment (stuffyAI) [reply core/assets/bundles/bundle_de.properties:721]: Joa das passt.

**Review by zenonet — CHANGES_REQUESTED** [review]
I think the sector names don't need a translation because it's not even necessary to know what these names mean and I at least think that most of them sound very weird. Some of them are ok like `Trümmerstrände` but `Extraktionsaußenposten`? I think these names are worse to read/speak than just not understanding them at all.

File: core/assets/bundles/bundle.properties [hunk 10]
```diff
@@ -705,22 +705,22 @@ planet.sun.name = Sun
...
+sector.craters.name = Die Krater
+sector.frozenForest.name = Gefrorener Wald
+sector.ruinousShores.name = Trümmerstrände
+sector.stainedMountains.name = Geflecktes Gebirge
+sector.desolateRift.name = Einsame Kluft
+sector.nuclearComplex.name = Kernherstellungskomplex
+sector.overgrowth.name = Überwucherung
+sector.tarFields.name = Teerfelder
+sector.saltFlats.name = Salzebenen
+sector.fungalPass.name = Infizierter Gebirgspass
```
Comment (zenonet) [inline core/assets/bundles/bundle.properties]: That's not even the correct translation of fungal in this context

**Comment by Catchears** [comment]
As I said in the original message, my issue was mostly that only some of the names had been translated. We could also leave them all English; I think the others' (@Shawak and @stuffyAI) opinions might be useful here.

**Comment by Shawak** [comment]
> As I said in the original message, my issue was mostly that only some of the names had been translated. We could also leave them all English; I think the others' (@Shawak and @stuffyAI) opinions might be useful here.

In my opinion translating names is usually a bad thing, but since mindustry is doing it everywhere anyways we should keep it as it is and translate them also.

**Comment by zenonet** [comment]
I think it would be fine to just revert the translations of all sector names including the ones that have been done before this PR

**Comment by Shawak** [comment]
I disagree, because then you would also want to not translate turret names etc.

**Comment by stuffyAI** [comment]
> I think it would be fine to just revert the translations of all sector names including the ones that have been done before this PR

You can look at all the other languages, any language that has somewhat active translations also has names translated, at least the non-new ones. I don't like it either, but we will should comply with the rest of the community and translate names.

**Comment by zenonet** [comment]
I guess, I am just going to continue playing in english.

File: core/assets/bundles/bundle.properties (hunk 10 above)
Comment (Catchears) [reply core/assets/bundles/bundle.properties]: I'd be open to suggestions, do you have a better idea?

File: core/assets/bundles/bundle_de.properties [hunk 11]
```diff
@@ -1152,7 +1153,7 @@ rules.coreincinerates = Kern verbrennt überflüssige Materialien
 rules.disableworldprocessors = Deaktiviere Weltprozessoren
 rules.schematic = Entwürfe erlaubt
 rules.wavetimer = Wellen-Timer
-rules.wavesending = Wave Sending
+rules.wavesending = Manuelle Wellen möglich
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1156]: Wellen-Senden oder

**Review by Shawak — CHANGES_REQUESTED** [review]

File: core/assets/bundles/bundle_de.properties (hunk 11 above)
Comment (Catchears) [reply core/assets/bundles/bundle_de.properties:1156]: Ja, sowas in der Art hatte ich auch überlegt. Am Ende hatte ich mich doch dagegen entschieden, da es sich nicht so anhören soll, als könnte man die Wellen dann ausschließlich manuell abschicken - wie siehst du das?

**Review by Shawak — APPROVED** [review]

**Comment by end-me-please** [comment]
maybe we should hurry up a bit + do the new v7 things before the release
the current translation situation is a bit messy

**Comment by Catchears** [comment]
what do you mean by hurry up?

**Comment by Anuken** [comment]
Is this ready to merge? Release is in under 24 hours...

**Comment by Catchears** [comment]
This is very nearly ready. I'll push 1 more commit in the next hour and then it'll be good to go.

**Pull request merged**